scipy/optimize/optimize.py

```diff
@@ -1998,6 +1998,11 @@ def brent(func, args=(), brack=None, tol=1.48e-8, full_output=0, maxiter=500):
     Uses inverse parabolic interpolation when possible to speed up
     convergence of golden section method.
 
+    Does not ensure that the located minimum is within the initially
+    specified bracketing interval. (See 'fminbound' to ensure the same)
```

Comment:
As @person142 already pointed out, this is still an incorrect use of "bracketing interval". There's 2 cases here:

1. ``brack`` is a tuple of length 2. In this case, ``brent`` uses ``bracket`` to provide a bracketing interval. The ``bracket`` docstring is informative, so just link to that rather than adding this notes section.
2. ``brack`` is a tuple of length 3. In this case those 3 values are used directly as the bracketing interval, and the returned solution should be within that interval.